dags/utils/gke.py

```diff
@@ -3,7 +3,7 @@ def create_gke_config(
     service_account,
     owner_label,
     team_label,
-    machine_type="n1-standard-1",
+    machine_type="n1-standard-2",
```

Comment:
Did you mean to change these settings?

Reply:
Yes, there are now two containers running on the same node, so I want each of them to get their own core.